engines/bastion_katello/app/assets/javascripts/bastion_katello/subscriptions/subscriptions.controller.js

```diff
@@ -39,11 +39,6 @@ angular.module('Bastion.subscriptions').controller('SubscriptionsController',
             $scope.transitionTo('subscriptions.index');
         };
 
-        $scope.groupedSubscriptions = {};
-        $scope.$watch('table.rows', function (rows) {
-            $scope.groupedSubscriptions = SubscriptionsHelper.groupByProductName(rows);
-        });
-
```

Comment:
Note difference when subscriptions are grouped by name. It is an important feature and quite common experience for customers. Let me know if you have other questions. Thanks!

![subscriptions master](https://cloud.githubusercontent.com/assets/338096/14917444/0c4dc8f0-0ded-11e6-9b38-e82abc566fd3.png)

![subscriptions](https://cloud.githubusercontent.com/assets/338096/14917443/0c4d6176-0ded-11e6-99c3-bee0326ad1d1.png)